angularSTEP/src/app/shopping-list/shopping-list.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { ShoppingListComponent } from './shopping-list.component';
+
+describe('ShoppingListComponent', () => {
```

Comment:
FWIW this component seems to be complicated enough to warrant more tests than this auto-generated placeholder provides

Reply:
I am actually going through and adding unit tests now for different aspects of components.